lang/common/bridge/avro/Generate-CSFromAvro.ps1

```diff
@@ -0,0 +1,55 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+$ReefRoot = [Environment]::GetEnvironmentVariable("REEFSourcePath")
```

Comment:
REEFSourcePath comes from the powershell setup on the REEF website: https://cwiki.apache.org/confluence/display/REEF/Windows+PowerShell+Setup. Currently this script is should only be run when the avsc file change. I can modify the script to use relative directory paths so that when it becomes a part of every build the environment variable is not needed.